### ISSUE TITLE
Improve Hajk's accessibility

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#877]
_This issue is a meta-issue that collects the fixes made on our journey towards greater accessibility._

Please add more items as you notice problems.

- [x] Enable the possibility to use keyboard for map navigation. Take a look at this example: https://openlayers.org/en/latest/examples/accessible.html. 
- [x] Setting `tabIndex="0"` to enable keyboard navigation in map is a solution. But I think that we also need to style the map element when it has focus, to indicate it for the user. I'm not sure how much styling is needed. This will probably need some discussion. 
- [x] Add missing `aria-label` where indicated by Lighthouse
- [x] Fix duplicate `id`s as indicated by Lighthouse
